### PR TITLE
[fpv/pinmux] Add testplan and README

### PR DESCRIPTION
**Pull request by cindychip**
Add testplan and a readme file for pinmux FPV tb

Signed-off-by: Cindy Chen <chencindy@google.com>

### CODE REVIEW

File: hw/ip/pinmux/data/pinmux_fpv_testplan.hjson [hunk 1]
```diff
@@ -0,0 +1,119 @@
+{
+  name: "pinmux"
+  import_testplans: ["hw/dv/tools/testplans/fpv_csr_testplan.hjson"]
+  entries: [
+    {
+      name: InSel0_A
+      desc: "When register periph_insel sets to 0, the corresponding mio_to_periph_o should be 0."
```
Comment (msfschaffner) [inline hw/ip/pinmux/data/pinmux_fpv_testplan.hjson:7]: I would maybe reword `sets` as `is set to` and use the word `must` instead of `should` since this is a strict requirement

File: hw/ip/pinmux/data/pinmux_fpv_testplan.hjson [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    {
+      name: OutSelN_A
+      desc: '''When register mio_outsel sets to any value between 3 and (3 + Number of periph out),
+      the corresponding mio_out_o should be equal to the related periph_to_mio_i value.'''
+      milestone: V2
+      tests: ["pinmux_assert"]
+    }
+    {
+      name: OutSelOOB_A
+      desc: ''''When register mio_outsel sets to any value larger than (3 + Number of periph out),
```
Comment (msfschaffner) [inline hw/ip/pinmux/data/pinmux_fpv_testplan.hjson:65]: looks like you have too many ticks here... and further below you seem to have too many ticks as well ;).

File: hw/ip/pinmux/data/pinmux_fpv_testplan.hjson (hunk 1 above)
Comment (cindychip) [reply hw/ip/pinmux/data/pinmux_fpv_testplan.hjson:7]: Thanks Michael, sounds good, will update the rest of the docs.

File: hw/ip/pinmux/data/pinmux_fpv_testplan.hjson (hunk 2 above)
Comment (cindychip) [reply hw/ip/pinmux/data/pinmux_fpv_testplan.hjson:65]: Thanks for the good catch Michael. Updated.
